packages/framework/src/utils/options.utils.ts

```diff
@@ -0,0 +1,7 @@
+export function resolveApiUrl(providedApiUrl?: string) {
+  return providedApiUrl || process.env.NOVU_API_URL || 'https://api.novu.co';
+}
+
+export function resolveSecretKey(providedSecretKey?: string) {
+  return providedSecretKey || process.env.NOVU_SECRET_KEY || process.env.NOVU_API_KEY || '';
+}
```

Comment:
Adding explicit return types to enforce the function boundary
```suggestion
export function resolveApiUrl(providedApiUrl?: string): string {
  return providedApiUrl || process.env.NOVU_API_URL || 'https://api.novu.co';
}

export function resolveSecretKey(providedSecretKey?: string): string {
  return providedSecretKey || process.env.NOVU_SECRET_KEY || process.env.NOVU_API_KEY || '';
}
```

Reply:
Done